src/lexer/lexer.cc

```diff
@@ -0,0 +1,35 @@
+#include "lexer/lexer.h"
+
+#include <fstream>
+#include <sstream>
+
+namespace lexer {
+
```

Comment:
Please consider avoiding blank lines at the beginning and end of namespace declarations.